src/components/context-test/context-provide/ContextPlugin.js

```diff
@@ -133,3 +146,19 @@ function getLocalConfigWithComponentProp (configs, componentName, propName) {
     return componentConfig && componentConfig.hasOwnProperty(propName)
   }) || undefined
 }
+
+// Just 2 levels deep merge. B has priority.
+export function mergeConfigs (configA, configB) {
```

Comment:
Can we use lodash merge or mergeWith?

Reply:
@ilearnjs nah. This one is more like merge strategy. We might change it to something else.